wire-grpc-client/build.gradle.kts

```diff
@@ -1,11 +1,11 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
Let's revert all changes to this file. It's a style tweak and an unused import so shouldn't be needed.